### PR TITLE
feat(command-env): add transfer subcommand

### PR DESCRIPTION
**Pull request by ashalfarhan**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

Related to: #3575

This PR will add ability to transfer environment variables from one site to another.
If you want to send env vars from one site to another you can run this sub command.
Given 2 arguments, the basic usage is:
- If both argument provided, then the first argument will be the sender, and the second one is the destination
- If only one argument provided, then that will be the destination, and the sender should be current directory site, otherwise it will exit with helpfull error message
- If there's no env from `siteA`, then it will exit because nothing to transfer

So users can use this command without open their browser to copy and paste the env vars from one to another

---

For us to review and ship your PR efficiently, please perform the following steps:

- [x] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [x] Read the [contribution guidelines](../CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.
- [x] Update or add tests (if any source code was changed or added) 🧪
- [x] Update or add documentation (if features were changed or added) 📝
- [x] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

koala5

### CODE REVIEW

**Comment by ashalfarhan** [comment]
After i run `npm run docs` this log occurs, is markdown magic trying to escape the slash?
image

**Comment by ashalfarhan** [comment]
I already run the npm run docs, but it seems the new subcommand doesn't added, am i missed something?

**Comment by ashalfarhan** [comment]
Hi @erezrokah, is there anything that i should change?

File: src/commands/env/transfer.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+EnvTransferCommand.description = `Transfer env from one site to another`
+
+EnvTransferCommand.args = [
+  {
+    name: 'siteA',
+    required: true,
+    description: '(From) site Id or Name, if second argument is ommited, this argument will be the destination (To)',
```
Comment (erezrokah) [inline src/commands/env/transfer.js:71]: Will this command work with a site name?

The API call accepts a site id.

**Review by erezrokah — CHANGES_REQUESTED** [review]
@ashalfarhan, very sorry for the late review.
This looks good, just one comment.

I'll need to do some manual testing and then we'll be good to go.

File: src/commands/env/transfer.js (hunk 1 above)
Comment (ashalfarhan) [reply src/commands/env/transfer.js:71]: Ah i forgot about that, let me fix that

Comment (ashalfarhan) [reply src/commands/env/transfer.js:71]: I dont think it's possible, since the `getSite` method required a site Id to be called.

Comment (erezrokah) [reply src/commands/env/transfer.js:71]: I think we could just change the description to remove the name.
In a future PR we could support site names too by listing all sites and matching the name (out of scope for this PR).

Comment (ashalfarhan) [reply src/commands/env/transfer.js:71]: Yeah i think so, can i update the docs?

Comment (erezrokah) [reply src/commands/env/transfer.js:71]: `npm run docs` should do that for you once you update the description in the code.
If that doesn't work, just push the code changes and I'll sync the docs

Comment (ashalfarhan) [reply src/commands/env/transfer.js:71]: Okay, u can test them manually

**Comment by ashalfarhan** [comment]
Hi @erezrokah, @lukasholzer.
Are you guys already trying to test this PR locally?

**Comment by lukasholzer** [comment]
@ashalfarhan sorry for the late check in on that one! We are currently on getting a huge refactoring over the finishing line today or tomorrow: https://github.com/netlify/cli/pull/3717 as we are going to remove the oclif framework from the CLI.

So I would like to get this over the line first before adding new features. As said the timeline for this will be today or tomorrow as we want to have a large enough window before the Christmas brake to test it properly.

If that is merged I could offer you to update the PR on the oclif removal and do then manual testing on it if you want so.

**Comment by ashalfarhan** [comment]
> @ashalfarhan sorry for the late check in on that one! We are currently on getting a huge refactoring over the finishing line today or tomorrow: #3717 as we are going to remove the oclif framework from the CLI.
> 
> So I would like to get this over the line first before adding new features. As said the timeline for this will be today or tomorrow as we want to have a large enough window before the Christmas brake to test it properly.
> 
> If that is merged I could offer you to update the PR on the oclif removal and do then manual testing on it if you want so.

Well, I'm creating another for PR #3876 
Closing this

**Pull request closed without merging**